### DISCUSSION TITLE
Tracking user journeys

### DISCUSSION THREAD

**Discussion by TomDoesb** [grafana/faro-web-sdk#356] [Q&A] [answered]
I've just started tracking user journeys and want to aggregate them to visualize popular paths taken through my web app. I'm struggling a bit to find out if this is possible. 
For reference the picture below from logrocket is an example of what im trying to achieve
![image](https://github.com/grafana/faro-web-sdk/assets/12149839/6114976e-4cb4-4939-a854-ceaf9a0cd364)

Is it possible to do this with faro and grafana / is it on the roadmap?

**Comment by codecapitano** [comment, marked as answer]
Hi @TomDoesb 

I think you already implemented (custom) events in Faro which define and track the journeys / the funnels?

Currently we do not provide a ootb visualization in the Frontend Observabillity plugin which visualizes journeys like in the example screenshot.

I think a feature like that is not on our roadmap at the moment. 


Cheers,
Marco

/cc @cedricziel
